core/services/fluxmonitor/flux_monitor.go

```diff
@@ -106,8 +106,9 @@ func (fm *concreteFluxMonitor) Start() error {
 	var wg sync.WaitGroup
 	err := fm.store.Jobs(func(j *models.JobSpec) bool {
 		if j == nil {
+			// FIXME: This seems like an invariant violation?
 			err := errors.New("received nil job")
-			logger.Error(err)
+			logger.Errorw("Unexpected nil job when fetching jobs", "error", err)
```

Comment:
I think in most places we call it simply `"err"` maybe use that here for consistency?